[2025-03-12T22:06:59.426+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-12T22:06:59.445+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_lake_pipeline.start_pipeline manual__2025-03-12T22:06:55.248425+00:00 [queued]>
[2025-03-12T22:06:59.454+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_lake_pipeline.start_pipeline manual__2025-03-12T22:06:55.248425+00:00 [queued]>
[2025-03-12T22:06:59.455+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-12T22:06:59.481+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): start_pipeline> on 2025-03-12 22:06:55.248425+00:00
[2025-03-12T22:06:59.488+0000] {standard_task_runner.py:72} INFO - Started process 173 to run task
[2025-03-12T22:06:59.501+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_lake_pipeline', 'start_pipeline', 'manual__2025-03-12T22:06:55.248425+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/tmp/tmp8gezh2e0']
[2025-03-12T22:06:59.504+0000] {standard_task_runner.py:105} INFO - Job 29: Subtask start_pipeline
[2025-03-12T22:06:59.550+0000] {logging_mixin.py:190} WARNING - /usr/local/lib/python3.10/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-12T22:06:59.629+0000] {task_command.py:467} INFO - Running <TaskInstance: data_lake_pipeline.start_pipeline manual__2025-03-12T22:06:55.248425+00:00 [running]> on host ***-scheduler
[2025-03-12T22:06:59.794+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_lake_pipeline' AIRFLOW_CTX_TASK_ID='start_pipeline' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T22:06:55.248425+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-12T22:06:55.248425+00:00'
[2025-03-12T22:06:59.800+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-12T22:06:59.834+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-12T22:06:59.836+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Pipeline démarré"']
[2025-03-12T22:06:59.859+0000] {subprocess.py:99} INFO - Output:
[2025-03-12T22:06:59.861+0000] {subprocess.py:106} INFO - Pipeline démarré
[2025-03-12T22:06:59.862+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-12T22:06:59.908+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-12T22:06:59.909+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_lake_pipeline, task_id=start_pipeline, run_id=manual__2025-03-12T22:06:55.248425+00:00, execution_date=20250312T220655, start_date=20250312T220659, end_date=20250312T220659
[2025-03-12T22:07:00.003+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-12T22:07:00.067+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-12T22:07:00.078+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
